services/IWorkFlow.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Metadata,
+  ProcessorRequest,
+  ProcessorResponse,
+  RouterData,
+  RouterResponse,
+  TransformationDefaultResponse
+} from "../types/types";
+
+export default interface IWorkFlow {
```

Comment:
IWorkflow -> DestinationService (Somehow calling this workflow seems bit odd)

Reply:
Shall we create a hierarchy for individual flows or a central hierarchy for all flows (destination flows, source, etc)?